### ISSUE TITLE
Dream DRM decoder starting issue in arch linux

### ISSUE THREAD

**Issue by guipan** [dvdesolve/pkgbuilds#1]
I cannot start dream in an updated arch linux.
It writes to me this error:
dream: error while loading shared libraries: libhamlib.so.2: cannot open shared object file: No such file or directory

What can I do? Thanks a lot in advance for help

**Comment by dimich-dmb** [comment]
Attempt to build `dream` with `hamlib 4.4-1` fails:
```
src/util/Hamlib.cpp: In member function ‘void CHamlib::LoadSettings(CSettings&)’:
src/util/Hamlib.cpp:426:14: error: call of overloaded ‘Put(const char [7], const char [13], rig_model_t&)’ is ambiguous
  426 |         s.Put("Hamlib", "hamlib-model", model);
...
src/util/Hamlib.cpp: In member function ‘void CHamlib::SaveSettings(CSettings&)’:
src/util/Hamlib.cpp:436:14: error: call of overloaded ‘Put(const char [7], const char [13], rig_model_t&)’ is ambiguous
  436 |         s.Put("Hamlib", "hamlib-model", iHamlibModelID);
...
make: *** [Makefile:2691: obj/Hamlib.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from src/GUI-QT/RigDlg.cpp:33:
src/GUI-QT/DialogUtil.h:51:13: error: conflicting declaration ‘typedef int rig_model_t’
   51 | typedef int rig_model_t;
      |             ^~~~~~~~~~~
In file included from /usr/include/hamlib/rig.h:38,
                 from src/GUI-QT/../util/Utilities.h:44,
                 from src/GUI-QT/RigDlg.h:31,
                 from src/GUI-QT/RigDlg.cpp:32:
/usr/include/hamlib/riglist.h:648:18: note: previous declaration as ‘typedef uint32_t rig_model_t’
  648 | typedef uint32_t rig_model_t;
      |                  ^~~~~~~~~~~
make: *** [Makefile:2731: obj/RigDlg.o] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by dvdesolve** [comment]
Should be fixed in [afdbd45](https://github.com/dvdesolve/pkgbuilds/commit/afdbd45d618ed734341cbc8d3e5f7ceb46b11feb)
